.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## Addresses Issue: 
```

Comment:
I this this should be: `## Resolves: ` in order to trigger the "closing keywords" functionality: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Reply:
I actually changed it (from the idaes-pse example I started with) here to avoid it automatically closing issues, as it seems more often than not PRs only address parts of existing issues rather than close them entirely.

It's entirely a judgement call on if that's actually true or not.  I only felt slightly that "Addresses" is better than "Resolves" IMO.